shared/cfg/guest-os/Linux/JeOS/22.x86_64.cfg

```diff
@@ -0,0 +1,25 @@
+- 22.x86_64:
```

Comment:
I don't plan to release it. I'm pushing JeOS.23 now, but not JeOS.22. Still, it doesn't hurt to help users generate their own, if they want to.